geometry_msgs/Point pt
---
bool s